packages/config/src/projects/bobanetwork.ts

```diff
@@ -107,16 +108,14 @@ export const bobanetwork: Project = {
       },
       exitMechanisms: [
         {
-          name: 'Regular exit',
-          description:
-            'When a user initiates a withdrawal it is processed as a L2 to L1 message. Because Boba Network is an optimistic rollup this transaction has to be included in a block and finalized. This takes several days to happen after which the funds can be withdrawn on L1.',
+          ...EXITS.REGULAR('optimistic', 'merkle proof'),
           references: [
             {
               text: 'The Standard Bridge - Boba documentation',
               href: 'https://docs.boba.network/developer-docs/bridging-l1-l2#the-standardtm-bridge',
             },
           ],
-          risks: [],
+          risks: [EXITS.RISK_CENTRALIZED_BLOCK_PRODUCER],
```

Comment:
Would be great to have a reference here.

Reply:
Anyone can publish stateBatch as long as they have a bond. Currently BondManager that checks this is mocked and vouch only for the OVM_Proposer EOA address

Reply:
We also don't have a bond manager listed for Boba.